qa/tasks/mgr/dashboard/test_mgr_module.py

```diff
@@ -125,38 +123,24 @@ def test_get(self):
                 }))
 
     def test_put(self):
-        self.set_config_key('config/mgr/mgr/telemetry/contact', '')
-        self.set_config_key('config/mgr/mgr/telemetry/description', '')
-        self.set_config_key('config/mgr/mgr/telemetry/interval', '72')
-        self.set_config_key('config/mgr/mgr/telemetry/leaderboard', 'False')
-        self.set_config_key('config/mgr/mgr/telemetry/organization', '')
-        self.set_config_key('config/mgr/mgr/telemetry/proxy', '')
-        self.set_config_key('config/mgr/mgr/telemetry/url', '')
-        self.set_config_key('config/mgr/mgr/telemetry/last_opt_revision', '2')
-        self.set_config_key('config/mgr/mgr/telemetry/enabled', 'True')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_level', 'critical')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_to_cluster', 'False')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_to_cluster_level', 'info')
+        self.set_config_key('config/mgr/mgr/test_orchestrator/log_to_file', 'True')
         self._put(
-            '/api/mgr/module/telemetry',
+            '/api/mgr/module/test_orchestrator',
             data={
                 'config': {
-                    'contact': 'tux@suse.com',
-                    'description': 'test',
-                    'last_opt_revision': 1,
-                    'enabled': False,
-                    'interval': 4711,
-                    'leaderboard': True,
-                    'organization': 'SUSE Linux',
-                    'proxy': 'foo',
-                    'url': 'https://foo.bar/report'
+                    'log_level': 'debug',
+                    'log_to_cluster': True,
+                    'log_to_cluster_level': 'warning',
+                    'log_to_file': False
                 }
             })
         self.assertStatus(200)
-        data = self._get('/api/mgr/module/telemetry')
+        data = self._get('/api/mgr/module/test_orchestrator')
         self.assertStatus(200)
-        self.assertEqual(data['contact'], 'tux@suse.com')
-        self.assertEqual(data['description'], 'test')
-        self.assertFalse(data['enabled'])
-        self.assertEqual(data['interval'], 4711)
-        self.assertTrue(data['leaderboard'])
-        self.assertEqual(data['organization'], 'SUSE Linux')
-        self.assertEqual(data['proxy'], 'foo')
-        self.assertEqual(data['url'], 'https://foo.bar/report')
+        self.assertEqual(data['log_level'], 'critical')
```

Comment:
I guess it needs to be:

```suggestion
        self.assertEqual(data['log_level'], 'debug')
```

http://pulpito.ceph.com/tdehler-2020-04-17_07:15:07-rados:dashboard-wip-tdehler-testing-34591-distro-basic-smithi/

```
2020-04-17T07:47:50.275 INFO:tasks.cephfs_test_runner:======================================================================
2020-04-17T07:47:50.275 INFO:tasks.cephfs_test_runner:FAIL: test_put (tasks.mgr.dashboard.test_mgr_module.MgrModuleTest)
2020-04-17T07:47:50.275 INFO:tasks.cephfs_test_runner:----------------------------------------------------------------------
2020-04-17T07:47:50.276 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2020-04-17T07:47:50.276 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_wip-tdehler-testing-34591/qa/tasks/mgr/dashboard/test_mgr_module.py", line 143, in test_put
2020-04-17T07:47:50.276 INFO:tasks.cephfs_test_runner:    self.assertEqual(data['log_level'], 'critical')
2020-04-17T07:47:50.276 INFO:tasks.cephfs_test_runner:AssertionError: u'debug' != 'critical'
```

Reply:
Fixed